src/power_grid_model/core/power_grid_model.py

```diff
@@ -353,55 +360,54 @@ def calculate_state_estimation(
-            output_component_types: list or set of component types you want to be present in the output dict.
-                By default all component types will be in the output
-            continue_on_batch_error: if the program continues (instead of throwing error) if some scenarios fails
-
-
-        Returns:
-            dictionary of results of all components
-                key: component type name to be updated in batch
-                value:
-                    for single calculation: 1D numpy structured array for the results of this component type
-                    for batch calculation: 2D numpy structured array for the results of this component type
-                        Dimension 0: each batch
-                        Dimension 1: the result of each element for this component type
-            Error handling:
-                in case an error in the core occurs, an exception will be thrown
+            symmetric (bool, optional): Whether to perform a three-phase symmetric calculation.
+
+                - True: Three-phase symmetric calculation, even for asymmetric loads/generations (Default).
+                - False: Three-phase asymmetric calculation.        
+            error_tolerance (float, optional): Error tolerance for voltage in p.u., applicable only when iterative=True.
+            max_iterations (int, optional): Maximum number of iterations, applicable only when iterative=True.
+            calculation_method (an enumeration): Use iterative linear method.     
+            update_data (dict, optional):
+                None: Calculate state estimation once with the current model attributes.
+                Or a dictionary for batch calculation with batch update. 
+                    key: Component type name to be updated in batch.
+
+                        - For homogeneous update batch (a 2D numpy structured array):
+
+                            - Dimension 0: Each batch.
+                            - Dimension 1: Each updated element per batch for this component type.
+                        - For inhomogeneous update batch (a dictionary containing two keys):
+
+                            - indptr: A 1D integer numpy array with length n_batch + 1. Given batch number k, the update array for this batch is
+                              data[indptr[k]:indptr[k + 1]]. This is the concept of compressed sparse structure.  
+                              https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_matrix.html
+                            - data: 1D numpy structured array in flat.     
+            threading (int, optional): Applicable only for batch calculation.
+
+                - < 0: Sequential
+                - = 0: Parallel, use number of hardware threads
+                - > 0: Specify number of parallel threads  
+            output_component_types ({set, list}, optional): List or set of component types you want to be present in the output dict. 
```

Comment:
shouldn't there be a whiteline before this line?

Reply:
Whiteline is used only before bullet points.